management/src/main/resources/META-INF/services/org.ehcache.spi.service.ServiceFactory

```diff
@@ -1 +1,2 @@
-org.ehcache.management.registry.DefaultManagementRegistryFactory
\ No newline at end of file
+org.ehcache.management.registry.DefaultManagementRegistryFactory
+org.ehcache.management.registry.DefaultSharedManagementServiceFactory
```

Comment:
Given what I understand of the shared management registry, having it declared here makes no sense.